### DISCUSSION TITLE
GITHUB, INC. Invoice

### DISCUSSION THREAD

**Discussion by axisrooms** [community/community#109657] [New to GitHub]
I have attached the payment receipt and have made the payment. I would like the invoice for this. can you please share the invoice for this.

**Comment by ThisIsMrIsmail** [comment]
Hi @axisrooms,
It seems like you may have reached out to the wrong place about your invoice. Here's what I recommend:

**Check Your Email:**
Invoices are usually sent by email after payment. Search your inbox (including spam) for anything from GitHub related to your subscription.

**GitHub Billing Page:**
If you purchased GitHub Pro or a similar paid service, go to the "Billing" section in GitHub "Settings" where you might find past invoices under the payment information or something.

**GitHub Support:**
If you still can't find your invoice, here's the contact link for GitHub Support: [https://support.github.com/contact](https://support.github.com/contact).

Have a Good Day, Peace.

**Reply by queenofcorgis** [reply]
@ThisIsMrIsmail 

We’ve clarified our stance on using generative AI tools like ChatGPT within our Community via [this announcement](https://github.com/orgs/community/discussions/69006). Please review the guidelines to ensure your post meets them as failure to adhere to those rules can result in action taken by our moderator team.  You can read our updated [Code of Conduct](https://github.com/community/community/blob/baa5cd9020ef3f2bbf86dca1191d9c90f9b0822d/CODE_OF_CONDUCT.md#reasonable-use-of-ai-generated-content) and the [announcement](https://github.com/orgs/community/discussions/69006) for more details. Thank you for helping us maintain an authentic and beneficial space for everyone.

**Reply by ThisIsMrIsmail** [reply]
Hi @queenofcorgis,
I am going to take it into consideration from now on.

Thank you for the clarification.
